lua/telescope/builtin/files.lua

```diff
@@ -209,6 +209,17 @@ files.find_files = function(opts)
     ),
     previewer = conf.file_previewer(opts),
     sorter = conf.file_sorter(opts),
+    attach_mappings = function(prompt_bufnr, map)
+
+			local change_working_directory = function ()
```

Comment:
I dont understand why its doing this weird indenting. it looks correct on my vim locally. `ts`, `sw` and `sts` are all set to 2

Reply:
Because you are using tabs and tabs will always have a width of 8 in github, it doesnt matter what you set in your local vim config.
We are using 2 spaces for indentation so `set expandtab`